kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunnerViewModel.kt

```diff
@@ -74,16 +72,27 @@ internal class WorkflowRunnerViewModel<OutputT : Any>(
         workflow,
         inputs,
         snapshot
-    ) { renderings, outputs ->
+    ) { renderings, output ->
       @Suppress("UNCHECKED_CAST")
-      WorkflowRunnerViewModel(viewRegistry, renderings, outputs, this) as T
+      WorkflowRunnerViewModel(renderings, output, this) as T
     }
   }
 
-  private val snapshotJob = scope.launch {
-    renderingsFlow
-        .map { it.snapshot }
-        .collect { lastSnapshot = it }
+  private val _result = BehaviorSubject.create<OutputT>()
```

Comment:
Use `SingleSubject`

Reply:
Or better yet, `scope.rxSingle`

Reply:
Btw looks like these builders are about to lose their scope receiver:

https://github.com/Kotlin/kotlinx.coroutines/commits/reactive-scopeless